WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -656,20 +656,14 @@ public Type PropertyType
     {
         get
         {
-            var type = Data?.GetType() ?? typeof(RedDummy);
+            var type = Data.GetType();
```

Comment:
this seems to have lost the reddummy?

Reply:
Since Data is never null, there's no need to fall back to RedDummy